cGame_logic.cpp

```diff
@@ -55,7 +56,7 @@ cGame::cGame() : m_timeManager(*this) {
     m_iniScreenWidth = -1;
     m_iniScreenHeight = -1;
 
-    m_version = "0.6.x";
+    m_version = D2TM_VERSION;
```

Comment:
Why move this to CMake?
Many libraries have their version in the source code. Usually a version.h or something.

Reply:
I don't know. Maybe it's easier than existing ?